units/generate_tasks.go

```diff
@@ -19,12 +19,25 @@ import (
 	"github.com/mongodb/grip/message"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/mongo"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
 const (
 	generateTasksJobName = "generate-tasks"
 )
 
+const (
+	taskIdAttribute                        = "evergreen.task.id"
+	taskVersionAttribute                   = "evergreen.task.version"
+	parseProjectSecondsAttribute           = "evergreen.parse_project.seconds"
+	mergeGeneratedProjectsSecondsAttribute = "evergreen.merge_generated_projects.seconds"
+	createNewVersionSecondsAttribute       = "evergreen.create_new_version.seconds"
+	validateConfigSecondsAttribute         = "evergreen.validate_config.seconds"
+	simulateDependenciesSecondsAttribute   = "evergreen.simulate_dependencies.seconds"
+	saveTasksSecondsAttribute              = "evergreen.save_tasks.seconds"
+)
```

Comment:
same comments as above about breaking up the strings

Reply:
I changed to not use these anymore (because OTEL already tells you how many seconds, that's kind of the point 😅 ) so the only attributes we'll need are the existing ones on [globals.go](https://gist.github.com/hadjri/993d8b2c9bd025a8c44be1c76d054b85#file-globals-go-L467-L477)